packages/x-charts-pro/src/ChartOverview/ChartOverview.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { useXAxes, useYAxes } from '@mui/x-charts/hooks';
+import { ChartAxisOverview } from './ChartAxisOverview';
+
+/**
+ * Renders the overview for all x and y axes that have an overview enabled.
+ */
+export function ChartOverview() {
```

Comment:
I think the name is not really descriptive. Do we plan to add other types of overview? This could be `ChartZoomOverview` or `ChartZoomSlider`

Reply:
> Do we plan to add other types of overview?

Yes, that was my rationale. As explained in https://github.com/mui/mui-x/issues/15383, there's the slider, which I'm calling "AxisOverview" here (maybe I should rename that?), but there's also a minimap, which basically is a 2D overview.

Reply:
Hum, I would probably make them separate components as they have very little in common

Reply:
Yeah, I think you might be right. They aren't exclusive either. You can have both if you want. I'll do that.